requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+flask==1.0.2
```

Comment:
See my comment about not using pip. And then this and requirements.txt can be removed.

Reply:
Yep! thanks